robottelo/performance/pulp.py

```diff
@@ -0,0 +1,140 @@
+"""Test utilities for writing Pulp tests
+
+Part of functionalities of Pulp are defined in this module
+and have utilities of single repository synchronization, single
+sequential repository sync, sequential repository re-sync.
+
+"""
+import logging
+
+from robottelo.cli.repository import Repository
+from robottelo.common import ssh
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Pulp(object):
+    """Performance Measurement of RH Satellite 6
+
+    Pulp Synchronization functionality
+
+    """
+    @classmethod
+    def repository_single_sync(cls, repo_id, repo_name, thread_id):
+        """Single Synchronization
+
+        :param str repo_id: Repository id to be synchronized
+        :param str repo_name: Repository name
+        :return: time measure for a single sync
+        :rtype: float
+
+        """
+        LOGGER.info(
+            'Synchronize {0} by thread-{1}:'
+            .format(repo_name, thread_id)
+        )
+
+        result = Repository.synchronize({
+            'id': repo_id
+        })
+
+        if result.return_code != 0:
+            LOGGER.error(
+                'Sync repository {0} by thread-{1} failed!'
+                .format(repo_name, thread_id)
+            )
+            return 0
+        LOGGER.info(
+            'Sync repository {0} by thread-{1} successful!'
+            .format(repo_name, thread_id)
+        )
+        return cls.get_elapsed_time(result.stderr)
+
+    @classmethod
+    def get_elapsed_time(cls, stderr):
+        """retrieve time from stderr"""
+        # should return only one time point as a single sync
+        real_time = ''
+        for line in stderr.split('\n'):
+            if line.startswith('real'):
+                real_time = line
+        return 0 if real_time == '' else float(real_time.split(' ')[1])
+
+    @classmethod
+    def _get_enabled_repos(cls, org_id):
```

Comment:
This method uses nothing on the class. Why bother making it a class method? How about making this a static method instead?